site/src/components/SignupForm/index.js

```diff
@@ -111,7 +111,7 @@ const SignupForm = () => {
                   id="firstname"
                   name="firstname"
                   maxLength="32"
-                  pattern="[A-Za-z]{1,32}"
+                  pattern="([A-Za-zŽžÀ-ÿ]+('[A-Za-zÀ-ÿ])?[A-Za-zŽžÀ-ÿ])|[[A-Za-zŽžÀ-ÿ]"
```

Comment:
I think there is some issue with the current one, I've added a ss in the comments. I'll recommend one more `/^[a-zA-Z'’\- ]+$/` here. Also after you finalize a regex could you please use these names to test it out , they don't cover everything but this should suffice, thanks

José Garcia-Hernández
Priya Patel-Joshi
Mohammed Al-Fayed
Yvonne D'Souza
Wei Zhang-Li
Evelyn O'Malley-Smith"

Reply:
Updated to a new regex that matches all the names you have given. This regex now accounts for presence of an apostrophe as well as a hyphen in between the name